patches/server/0009-Timings-v2.patch

```diff
@@ -697,6 +697,10 @@ index da922f395f0fff0881ead893c900c5b2623f48f0..1d03a79e9010bc514b72a81ba0ad4a62
 +    private static void timings() {
 +        boolean timings = getBoolean("timings.enabled", true);
 +        boolean verboseTimings = getBoolean("timings.verbose", true);
++        TimingsManager.url = getString("timings.url", "https://timings.aikar.co/");
```

Comment:
Put default as the variable

Reply:
I thought about doing that but decided against it for the same reason as all the other defaults do not use the defaults from the class fields directly either (some aren't even the same)

Reply:
They really should be the same; feels like that should rather be changed along with this.

Reply:
I think the options of the TimingsManager are the same but the ones from the Timings class aren't as that represents the not-initialized state of it hence them not being used in the config directly.

Another issue that I just noticed with directly using those variables from the manager would be that a reload would insert the previously configured ones as the defaults into the config (e.g. after removing the file) instead of re-generating the defaults like they are supposed to. I get that reloads aren't really supported but knowingly breaking it seems wrong to me.

Reply:
Good points! ✅ on my part, then